tests/v2/validation/rbac/rbac_test.go

```diff
@@ -0,0 +1,239 @@
+		errMessage := strings.Split(err.Error(), ":")[0]
+		assert.Equal(rb.T(),"namespaces is forbidden",errMessage)
+		fmt.Println("Validation successful!")	
+	}
+
+	//Testcase5 Validate if cluster members are able to list all the namespaces in a cluster
+	fmt.Printf("Testcase5 - Validating if %s can lists all namespaces in a cluster. \n",role)
+	//Get the list of namespaces as and admin client
+	namespaceListAdmin, err := GetListNamespaces(rb.client, rb.cluster.ID)
+	require.NoError(rb.T(), err)
+	//Get the list of namespaces as an admin client
+	namespaceListClusterMembers, err := GetListNamespaces(rb.standardUserClient, rb.cluster.ID)
+	if role == "cluster-owner"{
+		require.NoError(rb.T(), err)
+		//Length of namespace list for admin and cluster owner should match
+		assert.Equal(rb.T(), len(namespaceListAdmin), len(namespaceListClusterMembers))
+		//Namespaces obtained as admin and cluster owner should be same
+		assert.Equal(rb.T(), namespaceListAdmin, namespaceListClusterMembers)
+		fmt.Println("Validation successful!")
+	}else if role == "cluster-member"{
+		require.Error(rb.T(), err)
+		errMessage := strings.Split(err.Error(), ":")[0]
+		//assert namespaces list obtained as a cluster member is empty
+		assert.Equal(rb.T(), len(namespaceListClusterMembers), 0)
+		//assert cluster member is not able to list namespaces
+		assert.Equal(rb.T(), "namespaces is forbidden",errMessage)
+		fmt.Println("Validation successful!")
+	}
+
+	//Testcase6 Validate if cluster members are able to delete the project they are not owner of
+	fmt.Printf("Testcase6 - Validating if %s can delete a project they are not owner of. \n",role)
+	err = projects.DeleteProject(rb.standardUserClient, createProjectAsAdmin)
+	if role == "cluster-owner"{
+		require.NoError(rb.T(), err)
+		fmt.Println("Validation successful!")
+	}else if role == "cluster-member"{
+	require.Error(rb.T(), err)
+	errStatus := strings.Split(err.Error(), ".")[1]
+	rgx := regexp.MustCompile(`\[(.*?)\]`)
+	errorMsg := rgx.FindStringSubmatch(errStatus)
+	assert.Equal(rb.T(), "403 Forbidden" , errorMsg[1])
+	fmt.Println("Validation successful!")
+	}
+
+	//Testcase7 Validate if cluster members are able to delete the namespace they are not owner of
+	fmt.Printf("Testcase7 - Validating if %s can delete a namespace from a project they are not owner of. \n",role)
+	err = namespaces.DeleteNamespace(rb.standardUserClient,namespaceName, rb.cluster.ID )
+	if role == "cluster-owner"{
+		require.NoError(rb.T(),err)
+		fmt.Println("Validation successful!")
+	}else if role == "cluster-member"{
+		require.Error(rb.T(),err)
+		fmt.Println("Valdiation successful!")
+	}
+
+	//Testcase8 Validate if a standard user return nil list of clusters
+	//Remove added cluster member from the cluster as an admin
+	fmt.Println("Testcase8 - Validating cluster member is removed from the cluster and returns nil clusters")
+	err = users.RemoveClusterMember(rb.client, rb.standardUser)
+	time.Sleep(6 * time.Second)
```

Comment:
why 6 seconds? I think this should be using `wait.WatchWait` instead, if you're waiting on the user to remove?

Reply:
I agree use WatchWait there shouldn't be a reason to have a sleep

Reply:
Even after adding a WatchWait, test errors out as the user still has a list of clusters. Therefore added the time.Sleep. I'll look more into it probably using an incorrect resource wait.